packages/editor/src/lib/editor/Editor.ts

```diff
@@ -3070,18 +3106,16 @@ export class Editor extends EventEmitter<TLEventMap> {
 			this.off('tick', this._decayCameraStateTimeout)
 			this._cameraState.set('idle')
 			this.updateRenderingBounds()
+			this.updateHoveredId()
```

Comment:
We now need to update he hovered id when we are done with the camera movements. Otherwise interactions like scrolling might land us in a state where the cursor is on top of a shape, but the shape is not hovered.

Reply:
We would only need to do this when the select tool is active, right? We could also emit an event here that says "hey, we updated the camera bounds" and pick it up from the select tool. We did try pretty hard to split out the editor from the rest of tldraw (which includes our select tool, hence the update hovering shape id being in the tldraw package), but at this point I think it's just adding unnecessary complexity.